timelock-impl/src/test/java/com/palantir/atlasdb/timelock/batch/MultiClientConjureTimelockResourceTest.java

```diff
@@ -199,6 +220,8 @@ private AsyncTimelockService createAsyncTimeLockServiceForClient(String client)
                         .build()));
         when(timelockService.getCommitTimestamps(anyInt(), any()))
                 .thenReturn(Futures.immediateFuture(getCommitTimestampResponse(client)));
+        when(timelockService.unlock(any()))
```

Comment:
let's have this return the lock token so we can verify we correctly route requests to namespaces